File: 1.js


function simple(n) {

    const start = performance.now()

    let simple_arr = []

    for (let i = 2; simple_arr.length < n; i++) {

        let flag = false

        for (let j = 0; j < simple_arr.length; j++) {

            if ((i % simple_arr[j] === 0) && (i / simple_arr[j] !== 1)) {
                flag = true
                break
            }    
        }
        if (flag === false) {
            simple_arr.push(i)
        }
    }

    const end = performance.now()
    const time_ = (end - start)
    console.log(time_)
    
    return simple_arr
}

console.log(simple(process.argv[2]))

File: 1.py
import time

def simple(n):
    time_start = time.time()
    
    simple_n = []
    start = 2
    while len(simple_n) < n:
        
        flag = False
        
        for i in simple_n:
            if start % i == 0 and (start / i != 1):
                flag = True
                break
            
        if flag == False:
            simple_n.append(start)

        start += 1
            
    end = (time.time() - time_start)*1000
    print(end)
    return simple_n
        
print(simple(200))